src/plugins/irc-events/link.js

```diff
@@ -55,6 +55,8 @@ module.exports = function(client, chan, msg) {
 			head: "",
 			body: "",
 			thumb: "",
+			size: -1,
+			showSize: true,
```

Comment:
Can we perhaps combine this logic into just `size`? As far as I can see it's only being set to false for HTML content and errors. Keeping size at `-1` in these cases should be fine, and then only display size if its greater than zero.